### PR TITLE
chore(cli): let `env init` writes an environment manifest

### PR DESCRIPTION
**Pull request by Lou1415926**
Lets `env init` writes an initial environment manifest to `/copilot/environments/<env_name>/manifest.yml`, given user inputs (either from prompts or flags).

NOTE: depends on #3724. Marked "WIP" until #3724 is merged.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: internal/pkg/cli/env_init.go [hunk 1]
```diff
@@ -655,8 +659,45 @@ To recreate the environment, please run:
 	return nil
 }
 
-func (o *initEnvOpts) deployEnv(app *config.Application,
-	artifactBucketARN, artifactBucketKeyARN string) error {
+func (o *initEnvOpts) importVPCConfig() *config.ImportVPC {
```
Comment (Lou1415926) [inline internal/pkg/cli/env_init.go:659]: These two `importVPCConfig()` and `adjustVPCConfig()` are existing methods that were removed in #3724 for static check. No changes are made implementation-wise!

File: internal/pkg/cli/env_init.go [hunk 2]
```diff
@@ -187,6 +197,11 @@ func newInitEnvOpts(vars initEnvVars) (*initEnvOpts, error) {
 	}
 
 	prompter := prompt.New()
+
+	ws, err := workspace.New()
+	if err != nil {
+		return nil, fmt.Errorf("workspace cannot be created: %w", err)
```
Comment (iamhopaul123) [inline internal/pkg/cli/env_init.go]:
```suggestion
		return nil, fmt.Errorf("create workspace: %w", err)
```

**Review by iamhopaul123 — APPROVED** [review]
Do we consider moving the init process to `initialize` pkg 🤔

**Comment by Lou1415926** [comment]
> Do we consider moving the init process to initialize pkg 🤔

Yes this is on my to do list (and part of the refactor is already done). Right now I haven't done this because I think this is easier for reviewers to read the diff and be confident about the correctness.

**Review by efekarakus — APPROVED** [review]
yayay

**Comment by efekarakus** [comment]
added the label for PH's feedback

**Pull request merged**